Sources/PointFree/Account/Invite.swift

```diff
@@ -0,0 +1,158 @@
+import Css
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+let showInviteMiddleware =
+  requireTeamInvite
+    <| currentUserMiddleware
+    >-> writeStatus(.ok)
+    >-> respond(showInviteView)
+
+let revokeInviteMiddleware =
+  requireTeamInvite
+    <<< requireUser
+    <| { conn in
+      AppEnvironment.current.database.deleteTeamInvite(get2(conn.data).id)
+        .run
+        .flatMap(const(conn |> redirect(to: path(to: .team))))
+}
+
+let resendInviteMiddleware =
+  requireTeamInvite
+    <<< requireUser
+    <| { conn in
+      parallel(sendInviteEmail(invite: get2(conn.data), inviter: get1(conn.data)).run)
+        .run({ _ in })
+      return conn |> redirect(to: path(to: .team))
+}
+
+let acceptInviteMiddleware =
+  requireTeamInvite
+    <<< requireUser
+    <| { conn in
+
+      let sendInviterEmailOfAcceptance = parallel
+        <| AppEnvironment.current.database.fetchUserById(get2(conn.data).inviterUserId)
+          .run
+          .map(requireSome)
+          .flatMap { errorOrInviter in
+            errorOrInviter.right.map { inviter in
+              sendEmail(
+                to: [inviter.email],
+                subject: "\(get1(conn.data).name) has accepted your Point-Free team invitation!",
+                content: inj2(inviteeAcceptedEmailView.view((inviter: inviter, invitee: get1(conn.data))))
+                )
+                .run
+                .map(const(unit))
+              }
+              ?? pure(unit)
+      }
+
+      let deleteInvite = parallel
+        <| AppEnvironment.current.database.deleteTeamInvite(get2(conn.data).id).run
+
+      zip(sendInviterEmailOfAcceptance, deleteInvite).run({ _ in })
```

Comment:
just made an update to send an email to the inviter to let them know the invite was accepted, and then to delete the invite.

still need to add the current user to the inviter's subscription tho.